org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/scan/ScanRequest.java

```diff
@@ -77,17 +78,17 @@
 	 * scan command, chemical formula etc., grouped by type.
 	 */
 	private List<ScanMetadata> scanMetadata; // TODO use EnumMap instead of list?
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make "scanMetadata" transient or serializable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1948)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)